packages/tracks/src/index.js

```diff
@@ -14,8 +14,8 @@ const tracksDebug = debug( 'wc-admin:tracks' );
  * @param {string} eventName The name of the event to record, don't include the wcadmin_ prefix
  * @param {Object} eventProperties event properties to include in the event
  */
-
 export function recordEvent( eventName, eventProperties ) {
+	checkEventNameAndPropsCase( eventName, eventProperties );
```

Comment:
I'm a little hesitant to introduce this here for 2 reasons:

1. It may go unnoticed in the console.  If we can get a linting approach here, we can catch it while editing and prior to commit.
2. This appears to run in all environments (including non-dev environments).

Reply:
Let's split any checking/linting out to a separate PR, so we can get the fixes merged right away.

Reply:
Separating it out sounds good to me, we can further the discussion there. I was thinking about the linting approach a little more and not sure if that will work, given we pass the properties as variables sometimes in dynamically created objects, which I am not sure a linter can catch.